pulsar-client-cpp/wireshark/README.md

```diff
@@ -0,0 +1,84 @@
+# Pulsar Wireshark dissector
+
+The Pulsar Wireshark dissector allows to automatically decode the Pulsar binary protocol
+and visualize useful debug information (linking requests with responses, latency stats, etc.)
+
+### Install Wireshark
+
+#### macOS
+
+Recommend use this way to install Wireshark in macOS.
+
+```bash
+brew install homebrew/cask/wireshark
+```
+
+#### Ubuntu
+
+```bash
+sudo apt install wireshark
+```
+
+### Install dependencies
+
+Install Wireshark with the development headers, necessary in order to build the plugin.
+
+**NOTE**: Please make sure the Wireshark application version is the same as the Wireshark Headers version.
+
+#### macOS
+
+```shell
+$ brew install wireshark
+```
+
+#### Ubuntu
+
+```shell
+$ sudo apt install wireshark-dev
+```
+
+### Compile the dissector
+
+> If the compiler cannot find the wireshark headers,
+> please use `-DWIRESHARK_INCLUDE_PATH=<WIRESHARK_INCLUDE_PATH>` to manually add include path.
+
+Compile the dissector:
+
+```shell
+cd pulsar-client-cpp
+cmake -DBUILD_WIRESHARK=ON .
+make pulsar-dissector
+```
+
+This will create the `pulsar-dissector.so` plugin library in Wireshark directory.
+
+### Install Wireshark dissector
+
+Copy the dissector in the appropriate location so that Wireshark can find it at startup.
+
+#### Find the Personal Plugins Location
+
+1. Open Wireshark
+2. Click About Wireshark
```

Comment:
```suggestion
2. Click **About Wireshark**
```

Use bold for menus, menu items, separator symbols, items in the navigation tree, and the greater than symbol. If a menu contains variable menu items, use lowercase bold italic. 

https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.7x9g7hiebz8x